lms/djangoapps/certificates/apis/v0/views.py

```diff
@@ -0,0 +1,112 @@
+""" API v0 views. """
+import logging
+
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.generics import GenericAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework_oauth.authentication import OAuth2Authentication
+
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+
+from openedx.core.lib.api import permissions
+from lms.djangoapps.certificates.api import get_certificate_for_user
+
+log = logging.getLogger(__name__)
+
+
+class CertificatesDetailView(GenericAPIView):
+    """
+        **Use Case**
+
+            * Get the details of a certificate for a specific user in a course.
+
+        **Example Request**
+
+            GET /api/certificates/v0/certificates/{username}/{course_id}
```

Comment:
@giocalitri as this is a v0 API, I do think there is some flexibility wrt precisely following the conventions.  However if you have any flexibility wrt what's proposed in #11821 perhaps you would consider the following adjustment

```
# get all course certificates for a user
GET /api/certificates/v0/certificates/{username}/

# get course certificate for a user in a specific course
GET /api/certificates/v0/certificates/{username}/courses/{course_id}/
```

This minor change makes the URL a little clearer about the usage of the second identifier, and leaves room to introduce other sub-resources of the `/certificates/{username}/` collection in the future.  Not a dealbreaker, but aligns with recommended practices in most of the familiar sources.